### ISSUE TITLE
Yacas.gui installed from Snapcraft segfaults.

### ISSUE THREAD

**Issue by dirkmittler** [grzegorzmazur/yacas#346]
As the subject line suggests, I tried the Snapcraft version, just because it has a GUI, and it just segfaults.

Since the other bug which mentions a segfault has been closed, this must be a different bug.

Expected behaviour: GUI Opens
Actual behaviour: Just, a generic segfault. No error messages.

-Dirk

**Comment by deviantbit** [comment]
Yep. Also if you pull get it from their ppa and apt it crashes as well.
